dev-docs/bidders/dmx.md

```diff
@@ -13,8 +13,11 @@ gdpr_supported: true
 {: .table .table-bordered .table-striped }
 | Name       | Scope    | Description            | Example | Type     |
 |------------|----------|------------------------|---------|----------|
-| memberid | required | Represent boost MemberId from DistrictM UI | 'member1' | string |
-| tagid | optional | Represent the placement ID | '123abc' | string |
+| seller_id | required | Represent DMX Partner when you get onboarded, this is for specific setup BURL, NURL or ADM support | 'seller_id' | string |
+| publisher_id | required | Represent boost MemberId from DistrictM UI | 'member1' | string |
+| memberid | required | Represent boost MemberId from DistrictM UI *legacy replace by publisher_id| 'member1' | string |
+| tagid | required | Represent the placement ID from DistrictM or your own  | '123abc' | string |
+| dmxid | required | Represent the placement ID from DistrictM or your own *legacy replace by tagid | '123abc' | string |
```

Comment:
Don't understand this one... how is dmxid different than tagid? The comment seems to indicate that dmxid is "legacy, replaced by tagid", but it's still marked as "required". Is that correct?